Tools/CMake/AMReXConfig.cmake.in

```diff
@@ -44,7 +44,8 @@ set(AMReX_MPI_FOUND                 @AMReX_MPI@)
 set(AMReX_MPI_THREAD_MULTIPLE_FOUND @AMReX_MPI_THREAD_MULTIPLE@)
 set(AMReX_OMP_FOUND                 @AMReX_OMP@)
 set(AMReX_CUDA_FOUND                @AMReX_CUDA@)
-set(AMReX_DPCPP_FOUND               @AMReX_DPCPP@)
+set(AMReX_SYCL_FOUND                @AMReX_DPCPP@)
```

Comment:
I fear we use DPCPP/SYCL in a mix now - don't we want to stay with DPCPP for now?
(I am also ok to consistently rename it to SYCL.)

Reply:
`AMReX_DPCPP` is for internal use only and currently just a "legacy" variable that will be replaced at some point. I think it's fine to just call it SYCL.